src/test/editor-service_test.ts

```diff
@@ -198,6 +198,52 @@ function editorTests(editorFactory: (basedir: string) => EditorService) {
         });
   });
 
+  suite('getReferencesForFeatureAtPosition', function() {
+
+    test(
+        `it supports getting the references to ` +
+            `an element from its tag`,
+        async() => {
+          const contentsPath = path.join('editor-service', 'references.html');
+          const contents =
+              fs.readFileSync(path.join(basedir, contentsPath), 'utf-8');
+
+          await editorService.fileChanged(indexFile, `${contents}`);
+
+          deepEqual(
+              await editorService.getReferencesForFeatureAtPosition(
```

Comment:
Mind doing the WarningPrinter trick from https://github.com/Polymer/polymer-analyzer/blob/96d962492e201ccdaeac72923731642e2a96a9b3/src/test/html/html-element-reference-scanner_test.ts#L131 to make these asserts easier to read and verify?